app/utility/base_obfuscator.py

```diff
@@ -14,7 +14,7 @@ def run(self, link, **kwargs):
         supported_platforms = self.__getattribute__('supported_platforms')
         try:
             if agent.platform in supported_platforms and link.ability.executor in agent.executors:
-                link.command_hash = hashlib.md5(str.encode(link.command)).hexdigest()
+                link.command_hash = hashlib.sha256(str.encode(link.command)).hexdigest()
```

Comment:
this just seems to add to compute time for checking the uniqueness of completed commands

Reply:
running some tests it's about a 6% increase

Reply:
only a few ms difference